### PR TITLE
feat(cms): marketing archives integration

### PR DESCRIPTION
**Pull request by justneocity**
Merged marketing archives into contentful. Note: used development access token rather than preview (this ensures only published items are displayed, excluding draft) -> might need to change this for the other cms tables too.

### CODE REVIEW

File: frontend/pages/media/marketing.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ const MarketingContent = () => {
     const filteredContent = content.filter(
       (picture) =>
         selectedCategory === "All" ||
-        picture.category.split(",").includes(selectedCategory)
+        picture.category.includes(selectedCategory)
```
Comment (joanna209) [inline frontend/pages/media/marketing.js:89]: Is there a reason why we're removing the split here? For marketing archives, there is usually more than one category

File: frontend/pages/media/marketing.js [hunk 2]
```diff
@@ -42,37 +45,34 @@ const MarketingContent = () => {
...
 
-  // scroll to top on load
-  useEffect(() => window.scrollTo(0, 0), []);
-
   // get marketing archives
   // input: marketing archives data from google sheets
   // output: array of dictionaries containing marketing archives data
+  const fetchMarketingArchives = async () => {
+    const res = await getMarketingArchives();
+    const tempContent = humps.camelizeKeys(res);
```
Comment (joanna209) [inline frontend/pages/media/marketing.js:53]: Is there a reason why the year filtering methods (e.g. reverse() and filter() were removed for `tempContent`? Seems to be causing issues with the timeline

**Review by joanna209 — CHANGES_REQUESTED** [review]
A couple of things to address and some comments: 
- Please make a slack post for this ticket
- Some of the refactoring has caused issues with filtering, I've highlighted specific parts where these issues have occurred
- Write up some tests in the `__tests__` folder to check that we're getting the marketing data successfully and in the right order

File: frontend/pages/media/marketing.js [hunk 3]
```diff
@@ -42,37 +45,34 @@ const MarketingContent = () => {
...
+    console.log(res);
```
[the remark this answers is not in the document]
Comment (justneocity) [reply frontend/pages/media/marketing.js:54]: done

File: frontend/lib/Api.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { createClient } from "contentful";
+
+const client = createClient({
+  space: process.env.NEXT_PUBLIC_CONTENTFUL_API_SPACE,
+  accessToken: process.env.NEXT_PUBLIC_CONTENTFUL_API_TOKEN,
+  host: process.env.NEXT_PUBLIC_CONTENTFUL_API_HOST,
+});
+
+export async function loadMarketingArchives() {
```
Comment (joanna209) [inline frontend/lib/Api.js:9]: Delete `Api.js`

File: frontend/pages/media/marketing.js [hunk 5]
```diff
@@ -187,10 +179,10 @@ const MarketingContent = () => {
...
                         <Initiative
                           key={index}
-                          fb={content.link}
-                          imgUrl={content.imgUrl}
-                          alt={content.label}
-                          date={content.date}
+                          fb={content.fields.link}
+                          imgUrl={`https:${content.fields.img.fields.file.url}`}
+                          alt={content.fields.label}
+                          date={content.fields.date}
```
Comment (joanna209) [inline frontend/pages/media/marketing.js:185]: Dates should be formatted in DD/MM/YYYY format. Make sure to write tests!

File: frontend/pages/about/our-team.js [hunk 6]
```diff
@@ -71,10 +71,7 @@ export default function OurTeam({ execs, subcommittee }) {
   useEffect(() => {
     // show loading sign for team page
     setLoading(true);
-    const fetchDataPromises = [
-      fetchSubcom(subcommittee),
-      fetchExecs(execs),
-    ];
+    const fetchDataPromises = [fetchSubcom(subcommittee), fetchExecs(execs)];
```
Comment (joanna209) [inline frontend/pages/about/our-team.js:74]: Why was this changed?

File: frontend/pages/media/marketing.js [hunk 7]
```diff
@@ -45,34 +41,30 @@ const MarketingContent = () => {
...
   useEffect(() => {
-    window.scrollTo(0, 0);
-    // load marketing previews
-    fetchMarketingArchives().catch((error) =>
-      // error handling
-      console.error(error)
-    );
-  }, []);
-
+    setLoading(true);
```
Comment (joanna209) [inline frontend/pages/media/marketing.js:62]: Look into reimplementing the loading element for the timeline - it seems to be removed :(

**Review by joanna209 — CHANGES_REQUESTED** [review]

File: frontend/pages/about/our-team.js (hunk 6 above)
Comment (Eddyyyh) [reply frontend/pages/about/our-team.js:74]: Prettier did that ^

**Review by joanna209 — APPROVED** [review]

**Pull request merged**